### ISSUE TITLE
keyword LIKE is not parsed as a comparison 

### ISSUE THREAD

**Issue by DavidVentura** [andialbrecht/sqlparse#493]
The query
`'select 1 from t where a=1 and b like "%2%"'`

gives, for the `where` group:

```
[<Keyword 'where' at 0x7F3404F0A468>,
<Comparison 'a=1' at 0x7F3404F078B8>,
<Keyword 'and' at 0x7F3404F0A6A8>,
<Identifier 'b' at 0x7F3404F07930>,
<Keyword 'like' at 0x7F3404F0A828>,
<Identifier '"%2%"' at 0x7F3404F079A8>]
```
I'd expect `b like %2%` to be a comparison. Is this intenteded?

**Comment by dbczumar** [comment]
@andialbrecht Are there any updates on this? I'd be more than happy to file a PR to parse `like` and `ilike` as comparisons.

**Comment by dbczumar** [comment]
@andialbrecht I've filed https://github.com/andialbrecht/sqlparse/pull/525 to address this issue. Please let me know if you can take a look!

**Comment by andialbrecht** [comment]
Fixed in 667aee971aedfca140be8c553f683d831adfc927

Thanks @dbczumar !
